hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Item from "./Item";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+describe("InventoryItem", () => {
+    const image =
+        "hackathon_site/dashboard/frontend/src/assets/images/testImages/IO6e5a6.jpg";
```

Comment:
All of the images we'll be using is coming from a link. I only uploaded 3 so far for the purpose of testing. set `const image` to any one of these:
- https://i.imgur.com/iUpI1hC.jpg
- https://i.imgur.com/IO6e5a6.jpg
- https://i.imgur.com/kOlrXto.jpg

Delete the 3 images you've uploaded into assets/images/testImages

Reply:
I asked Graham and he said to go ahead and add the images into the folders though
Since I think he said that he wants to avoid having to use the internet for the tests?

Reply:
hm okay keep them then